### PR TITLE
Circuralar Disque Multi-Color

### PR DESCRIPTION
**Pull request by ReneDev01**
I reviewed my code, I removed the JS and I reviewed my css. And include.js

### CODE REVIEW

File: Art/ReneDev01/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../ReneDev01/style.css">
```
Comment (LaurelineP) [inline Art/ReneDev01/index.html:6]: # Feedback - incorrect style file path
Issue - this is related to understanding relative path here.
your path to the css file is pointing to a parent folder that does not exist in your PR
Here is a resource that might help you understand how to link a css file
https://www.w3schools.com/html/html_links.asp

# Context
```html
  <link rel="stylesheet" href="../ReneDev01/style.css">
```

File: Art/ReneDev01/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <script>
+        // Pour masquer le chargement après un certain temps (par exemple, 5 secondes)
+        setTimeout(function() {
+           /*  var loader = document.querySelector('.loader');
+            loader.style.display = 'none'; */
+            var multicolorLoader = document.querySelector('.multicolor-loader');
+            multicolorLoader.style.display = 'none';
+        }, 40000); // 5000 millisecondes (5 secondes)
+    </script>
+</body>
```
Comment (LaurelineP) [inline Art/ReneDev01/index.html:21]: # Feedback - JS not allowed
As the README.md states - the contributions should not have any JS 
Please adjust your contribution in order for your animations to be only HTML and CSS related.
The only JS modification that are authorised are the only described contribution to add in `include.js`
- [ ] Please remove any JS content on that file


# Context
```html

    <script>
        // Pour masquer le chargement après un certain temps (par exemple, 5 secondes)
        setTimeout(function() {
           /*  var loader = document.querySelector('.loader');
            loader.style.display = 'none'; */
            var multicolorLoader = document.querySelector('.multicolor-loader');
            multicolorLoader.style.display = 'none';
        }, 40000); // 5000 millisecondes (5 secondes)
    </script>
```

File: Art/ReneDev01/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        /* .loader {
+            border: 16px solid transparent;
+            border-top: 16px solid #f3f3f3;
+            border-radius: 50%;
+            width: 120px;
+            height: 120px;
+            animation: spin 2s linear infinite;
+        } */
```
Comment (LaurelineP) [inline Art/ReneDev01/style.css:19]: # Feedback - unnecessary comment 
[ ] - please remove the comments 

## Context 
```css
     /* .loader {
            border: 16px solid transparent;
            border-top: 16px solid #f3f3f3;
            border-radius: 50%;
            width: 120px;
            height: 120px;
            animation: spin 2s linear infinite;
        } */
```

File: include.js [hunk 4]
```diff
@@ -1978,6 +1978,13 @@ let cards = [
     author: 'Ishan', 
     githubLink: 'https://github.com/ishanchaskar' 
   },
+  {
+    artName: 'Circular multi-color Disk',
+    pageLink: './Art/ReneDev01/index.html', 
+    imageLink: './Art/ReneDev01/Disk-2023-10-18-09-38-09',
```
Comment (LaurelineP) [inline include.js:1984]: # Feedback - incorrect `imageLink`
- [ ] Please add the `.gif` extension to correctly refer to your  animated file
## Context
```js
 imageLink: './Art/ReneDev01/Disk-2023-10-18-09-38-09', 
```

**Review by LaurelineP — CHANGES_REQUESTED** [review]

**Comment by ReneDev01** [comment]
thank you for your feedback

File: include.js (hunk 4 above)
Comment (ReneDev01) [reply include.js]: I correct it.

**Review by LaurelineP — APPROVED** [review]

**Comment by LaurelineP** [comment]
Note to my peers:
Resolved the conflicts manually
- too many conflicts
- some contributions in include.js were duplicated ( manually checked ) hence I removed some

@ReneDev01 Thanks for your contribution - nice too see some french :) 
Note: when you do contribute on project where the language is english, do not forget to follow the repository / project already established choices ( here would be to stick in english )

Happy hacktoberfest 🙌

**Pull request merged**